application-services/custom/camera-management/appcamera/routes.go

```diff
@@ -257,6 +254,19 @@ func (app *CameraManagementApp) ptzRoute(w http.ResponseWriter, req *http.Reques
 	var res dtosCommon.BaseResponse
 	var err error
 
+	// Get pan/tilt x,y range for a particular device
+	ptzConfigs, err := app.getPTZConfiguration(deviceName)
+	if err != nil {
+		respondError(app.lc, w, http.StatusInternalServerError,
+			fmt.Sprintf("Failed to get PTZ configuration for the device %s: %v", deviceName, err))
+		return
+	}
+	// Offset x,y max/min values by 5%(or some small percentage)
+	left := panTiltOffset * ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.XRange.Min
+	right := panTiltOffset * ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.XRange.Max
+	up := panTiltOffset * ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.YRange.Max
+	down := panTiltOffset * ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.YRange.Min
```

Comment:
`panTiltOffset` should actually be multiplied by the range and not the min/max, like so:

```suggestion
	xRange := ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.XRange.Max - ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.XRange.Min
	right := panTiltOffset * xRange
	left := -right

	yRange := ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.YRange.Max - ptzConfigs.PTZConfiguration[0].PanTiltLimits.Range.YRange.Min
	up := panTiltOffset * yRange
	down := -up
```